authors/apps/authentication/views.py

```diff
@@ -11,6 +14,26 @@
 
 
 class RegistrationAPIView(APIView):
+    schema = AutoSchema(manual_fields=[
```

Comment:
Thanks for the question, I'll make sure to add a brief comment to explain what it does.

The `AutoSchema` class enables us to dictate the generation of the input fields, thus enabling us to make requests from the generated swagger doc, We had chosen to use it because the registration endpoint inherits from the `APIView` which can be easily linked to AutoSchema via the `schema` property of the `APIView` class. 

More info on understanding it can be found [here](https://www.django-rest-framework.org/api-guide/schemas/#autoschema)

Reply:
@jomasim I made some changes and this is no longer used. You can review again to see the updated file structure.